docs/en-us/dev/SPI.md

```diff
@@ -189,12 +189,12 @@ public interface Transporter {
 }
 ```
 
-for the method bind(),Adaptive will firstly search `server` key，if no Key  were founded then will search `transport` key ，to determine the implementation that the proxy represent for.
+for the method bind(), Adaptive will firstly search `server` key, if no Key were founded then will search `transport` key, to determine the implementation that the proxy represent for.
```

Comment:
```suggestion
For the method `bind()`, Adaptive will firstly search `server` key, if no key was found, it will search `transport` key, to determine the implementation which proxy represent for.
```

Reply:
The `Key` represent for `Key` in `Dubbo use URL (containing Key-Value) to pass the configuration.`, so it should be `Key`.

Reply:
> The `Key` represent for `Key` in `Dubbo use URL (containing Key-Value) to pass the configuration.`, so it should be `Key`.

I am very sorry I was careless, thank you for correcting